lib/src/components/NxDonutChart/NxDonutChart.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent, ReactElement } from 'react';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxDonutChart: FunctionComponent<Props> =
+  function NxDonutChart(props): ReactElement<Props> {
```

Comment:
Use `forwardRef` so that tooltips could be attached.  See `NxButton` for an example.

Reply:
> Use `forwardRef` so that tooltips could be attached. See `NxButton` for an example.

Changed to use `forwardRef` in https://github.com/sonatype/sonatype-react-shared-components/commit/47355af1906408054b5f20f7a7b2cea5319f34f5

Reply:
> You still need to actually attach the ref to the `<svg>`

Should be done with https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/5a3e4fd020df42fb5582f6d3dd79160c5e40120b